setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
```

Comment:
`setup.py` is on its way out. I recommend `pyproject.toml`.

Reply:
I replaced it with a `pyproject.toml` in a new PR, thanks.